### PR TITLE
Automated Contributors List with Scroller Added to README 📜✨

### LINKED ISSUE #1953: Automate Contributors List and Add Scroller to README
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/ajay-dhangar/algo/blob/main/CONTRIBUTING.md).

### Description
The current README has a manually updated contributors list, which requires frequent updates and makes the README appear bulky. To improve this, we need to:

1. **Automate the contributors list** so it dynamically updates whenever a new contributor joins.
2. **Add a horizontal scroller** to display contributors in a compact, visually appealing way.

### Goals
- **Dynamically update contributors** in the README to avoid manual maintenance.
- **Implement a scroller** for the contributors section to keep the README organized and prevent it from being overcrowded.

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by neeru24**
This PR integrates the All Contributors Bot for dynamic contributor updates and implements a horizontal scroller to display contributors in a compact format. These changes streamline maintenance and enhance readability in the README.

fixes: #1953 
@ajay-dhangar 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,1265 +170,15 @@ We welcome contributions from developers of all experience levels. For guideline
...
-                    <br />
-                    <sub><b>Amankr200</b></sub>
-                </a>
-            </td>
-		</tr>
-	<tbody>
-</table>
+<p align="center">
+    <img src="https://api.vaunt.dev/v1/github/entities/ajay-dhangar/repositories/algo/contributors?format=svg&limit=54" width="700" height="250" />
+</p>
```
Comment (ajay-dhangar) [inline README.md]: replace

```md
<p align="center">
    <img src="https://api.vaunt.dev/v1/github/entities/ajay-dhangar/repositories/algo/contributors?format=svg&limit=54" width="700" height="250" />
</p>
```

with

```md
<p align="center">
    <img src="https://api.vaunt.dev/v1/github/entities/ajay-dhangar/repositories/algo/contributors?format=svg&limit=54" width="1000" />
</p>
```

File: README.md [hunk 2]
```diff
@@ -170,1265 +170,15 @@ We welcome contributions from developers of all experience levels. For guideline
...
+
+<p align="center">
+<a href="https://github.com/ajay-dhangar/algo/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ajay-dhangar/algo" />
+</a>
+</p>
```
Comment (ajay-dhangar) [inline README.md]: Delete lines 177 to 181.

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by neeru24** [comment]
@ajay-dhangar DONE WITH THE CHANGES

**Review by ajay-dhangar — APPROVED** [review]
Done and everything is fine.

**Pull request merged**